src/constants/typography.js

```diff
@@ -149,6 +149,44 @@ const Connect = styled.a`
   }
 `
 
+const FooterLink = styled.a`
+  font-size: ${fontSizes.p};
+  font-family: ${fonts.navButton};
```

Comment:
I think the font for the links might be a little too bold from the designs? I tried doing `fonts.button` with letter spacing 0.5 and it looks more like what they have:
<img src="https://user-images.githubusercontent.com/60420625/117349299-3d70a380-ae79-11eb-97a2-11acd1ad2f88.JPG" alt="footer links">
We can also increase the mobile line-height to around 24px
<img src="https://user-images.githubusercontent.com/60420625/117349307-3ea1d080-ae79-11eb-9db0-d5619803a395.JPG" alt="mobile footer links" />